README.md

```diff
@@ -78,19 +78,34 @@ which we feel adds to the complexity.
 - corruption of the git-node with locks in. Delete them and end the git-depend
 
 # Quick demo
+
+`git-depend` should be built with `go version` >= `1.16`
```

Comment:
Doesn't it build with at least `1.15`?

Reply:
It should do but I only tested it with `1.16`. Happy to switch that back to `1.15` if you would prefer.

Reply:
https://github.com/git-depend/git-depend/blob/main/go.mod#L3

Just rewrite it as 1.15. I'm kinda surprised go doesn't have a version checker that warns you about this tbh.

Reply:
Done.